pkgs/development/python-modules/pandoc-attributes/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchPypi
+, pandocfilters
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-attributes";
+  version = "0.1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "69221502dac74f5df1317011ce62c85a83eef5da3b71c63b1908e98224304a8c";
+  };
+
+  buildInputs = [
+    pandocfilters
```

Comment:
why isn't this part of `propagatedBuildInputs`? It's also needed at runtime.

Reply:
How can I check what needs to be in 'propagatedBuildInputs'?

Reply:
Check `setup.py`. `install_requires` goes to `propagatedBuildInputs`, `setup_requires` to `nativeBuildInputs` and `test_require` to `checkInputs`.